### DISCUSSION TITLE
Single MR / multiple commits for all dependencies

### DISCUSSION THREAD

**Discussion by LVerneyPEReN** [renovatebot/renovate#13587] [Request Help] [answered]
Hi,

I am looking for a way to have a single MR but multiple commits for new versions with Renovate.

As of now, I have `"groupName": "all"` which handles the single MR. It however generates a single commit containing all of the updates, not letting me cherry-pick easily (therefore a partial merge could be intepreted in Renovate as a rejection of the specific updates).

Typically, today, I got a MR created with a single commits and 4 proposed updates. 3 of them are ok, one of them is breaking. I'm looking for an easy way to merge the 3 OK updates and leave the last one aside. In the multiple MR scheme, I would merge all but one MR and close the last one. Renovate would catch it and stop notifying about this specific update. In the single MR scheme, I'm missing this possibility to reject a single update while merging the rest.

Reason for having a single MR scheme is that I have a special use case where most of my dependencies are intricated and I usually just want to merge updates as soon as possible / minimum number of clicks.

Is there a way to achieve such a workflow in Renovate?

Thanks!

**Comment by rarkins** [comment, marked as answer]
This is not possible
